code/game/objects/systems/cell_slot.dm

```diff
@@ -101,3 +134,83 @@
 	obj_cell_slot.remove_yank_time = 0
 	obj_cell_slot.legacy_use_device_cells = TRUE
 	return obj_cell_slot
+
+//? Wrappers for cell.dm functions
+//+ These exist to sanely perform cell functions through the obj_cell_slot system. These break safely if no cell exists.
+
+/**
+ * cell function wrapper - returns the rating of the cell inside or null
+ */
+/datum/object_system/cell_slot/proc/get_rating()
+	return cell?.get_rating()
+
+/**
+ * cell function wrapper - returns the cell src or null
+ */
+/datum/object_system/cell_slot/proc/get_cell(inducer)
```

Comment:
```suggestion
/datum/object_system/cell_slot/proc/get_cell(var/inducer)
```
While not required, once again improves readability, and clarifies what type inducer is (I am assuming bool in this case)

Reply:
we specifically do not use var/ anywhere in parameters because it's implicit and therefore unnecessary